### DISCUSSION TITLE
Postfix - SMTP is blocked my digital ocean server can i use a 3rd party Mail server

### DISCUSSION THREAD

**Discussion by akolaarthurali** [pretix/pretix#4095] [Q&A]
I recently setup pretix on a digital ocean VM droplet, I install postfix too, but to my surprise postfix is blocked on digital ocean (Due to spam issue, just as their 2022 policy states) so my inquiry is whether I can use any other 3rd- party mail server other Postfix  like https://forwardemail.net/en and if not seek your guidance on how I configure postfix to run on another port successfully 

![Screenshot_20240419_144900_Chrome](https://github.com/pretix/pretix/assets/34062229/0fa57adb-cdc3-45a6-92ce-153c0d15a148)

**Comment by raphaelm** [comment]
When looking for a third-party service, we recommend going with a service that advertises itself as "transactional mail" or "SMTP gateway", these can usually deal with the volume.
